bin/wmagent-mod-config

```diff
@@ -343,7 +346,7 @@ def main(argv=None):
                           '--ufc_port', '--ufc_service_url', '--ufc_cachedir',
                           '--client_mapping', '--workload_summary_url',
                           '--wmstats_url', '--ops_proxy', '--reqmgr2_url', '--acdc_url',
-                          '--amq_auth_file', '--dbs3_url', '--dqm_url',
+                          '--amq_auth_file', '--dbs3_url', 'dbs3_reader_url', '--dqm_url',
```

Comment:
I am almost sure you have to define it like `--dbs3_reader_url` instead of `dbs3_reader_url` (missing double dashes)

Reply:
thanks for spotting this, fixed.

Reply:
Also missing.